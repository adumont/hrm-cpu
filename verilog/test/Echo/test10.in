51 b7
07 17
02 87
19 8c
38 11
45 07
18 d3
20 a4
28 df
08 69
59 90
31 98
24 56
14 6c
15 35
55 fc
21 8c
50 62
07 85
38 06
01 19
15 f2
40 cd
49 63
51 bc
11 0a
04 55
21 11
51 06
52 7a
41 4f
40 a9
15 7a
03 68
29 9d
20 18
