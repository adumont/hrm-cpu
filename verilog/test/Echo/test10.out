b7
17
87
8c
11
07
d3
a4
df
69
90
98
56
6c
35
fc
8c
62
85
06
19
f2
cd
63
bc
0a
55
11
06
7a
4f
a9
7a
68
9d
18
